package.json

```diff
@@ -12,6 +12,7 @@
   "author": "Osita, Erika, J, Heriberto",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
Would recommend in future that package addition is done in separate branches with a PR of its own.

Reply:
Also, I believe `axios` is already installed on our project. If you look at the `package.json` on the master branch you'll see that's there.